### PR TITLE
Tickets/dm 38158

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: python/lsst/ts/ataos/ataos_csc.py [hunk 1]
```diff
@@ -1140,13 +1135,10 @@ async def do_disableCorrection(self, data: type_hints.BaseMsgType) -> None:
...
         await asyncio.sleep(0.0)  # give control back to event loop
 
-        # Lower mirrors if appropriate
-        if data == "disableAll":
-            await self.lower_mirrors_to_hardpoints(m1=True, m2=True)
-        elif data == "m1":
-            await self.lower_mirrors_to_hardpoints(m1=True, m2=False)
-        elif data == "m2":
-            await self.lower_mirrors_to_hardpoints(m1=False, m2=True)
+        await self.lower_mirrors_to_hardpoints(
```
Comment (patrickingraham) [inline python/lsst/ts/ataos/ataos_csc.py:1138]: This would benefit from a unit test if possible. However, release and deploy an alpha for the time being.

**Review by patrickingraham — CHANGES_REQUESTED** [review]

File: python/lsst/ts/ataos/ataos_csc.py (hunk 1 above)
Comment (tribeiro) [reply python/lsst/ts/ataos/ataos_csc.py:1140]: I updated the enable correction unit tests to also check the disable correction. Also took the opportunity to make a few additional updates to the CSC to future proof it against some features marked for deprecation in salobj.

File: doc/conf.py [hunk 2]
```diff
@@ -2,9 +2,8 @@
 This configuration only affects single-package Sphinx documentation builds.
 """
 
-from documenteer.conf.pipelinespkg import *  # noqa
-
 import lsst.ts.ataos  # noqa
+from documenteer.conf.pipelinespkg import *  # noqa
```
Comment (patrickingraham) [inline doc/conf.py:6]: what is the #noqa?

File: python/lsst/ts/ataos/ataos_csc.py [hunk 3]
```diff
@@ -1521,7 +1514,7 @@ async def publish_enable_corrections(self) -> None:
             moveWhileExposing=self.move_while_exposing, **kwargs
         )
 
-    def shutter_monitor_callback(self, data: type_hints.BaseMsgType) -> None:
+    async def shutter_monitor_callback(self, data: type_hints.BaseMsgType) -> None:
```
Comment (patrickingraham) [inline python/lsst/ts/ataos/ataos_csc.py:1517]: all functions are async by default now?

**Review by patrickingraham — APPROVED** [review]
Thanks for adding the test (and the other updates)
Asked one question.

**Comment by patrickingraham** [comment]
Note your tests are failing...so clearly that needs to be addressed before merging.

File: python/lsst/ts/ataos/ataos_csc.py (hunk 3 above)
Comment (tribeiro) [reply python/lsst/ts/ataos/ataos_csc.py:1517]: Callbacks yes... they need to be async.

File: doc/conf.py (hunk 2 above)
Comment (tribeiro) [reply doc/conf.py:6]: I think it's to ignore the checks in this line. I think it's because `import *` violates some flake8 check.

**Pull request merged**